index.html

```diff
@@ -31,28 +32,232 @@ <h1 class="display-4 fw-bolder">UMKM REJOSO</h1>
-                            <img class="card-img-top" src="" alt="Product Image" height="250px">
+                            <img class="card-img-top" src="assets/image/rempeyekBuNanik.jpg" alt="Product Image" height="250px" width="270px">
                             <div class="card-body p-4">
                                 <div class="">
                                     <h6 
                                         style="color: darkgrey;">
                                         <!-- ICON BELUM TAMPIL -->
                                         <i class="bi bi-shop"></i>&nbsp;Rempeyek bu nanik
                                     </h6>
-                                        Harga mulai 2000 (bisa req harga)
+                                        Harga mulai 2000 (bisa req harga).
                                         <br>Varian : kacang tanah, udang rebon, kacang kedelai
                                 </div>
                             </div>
                             <div class="card-footer p-4 pt-0 border-top-0 bg-transparent">
                                 <div>
-                                    <a class="btn btn-outline-dark mt-auto" href="https://wa.me/6285706061089">
+                                    <a class="btn btn-outline-success mt-auto" href="https://wa.me/6285706061089">
+                                        <!-- ICON BELUM TAMPIL -->
+                                        <i class="bi bi-whatsapp"></i>&nbsp;Beli Sekarang
+                                    </a>
+                                </div>
+                                <br>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col mb-5">
+                        <div class="card h-100">
+                            <img class="card-img-top" src="assets/image/cateringBuWulan2.jpg" alt="Product Image" height="250px" width="270px">
+                            <div class="card-body p-4">
+                                <div class="">
+                                    <h6 
+                                        style="color: darkgrey;">
+                                        <!-- ICON BELUM TAMPIL -->
+                                        <i class="bi bi-shop"></i>&nbsp;Catering Makanan Bu Wulan
+                                    </h6>
+                                        Melayani catering nasi kotak, tumpeng, catering hajatan dll.<br>
+                                        Nasi kotak harga : 20.000-27.000<br>
+                                        Tumpeng : 250.000-500.000                                    
+                                </div>
+                            </div>
+                            <div class="card-footer p-4 pt-0 border-top-0 bg-transparent">
+                                <div>
+                                    <a class="btn btn-outline-success mt-auto" href="https://wa.me/6285330139929">
```

Comment:
https://stackoverflow.com/questions/15551779/open-link-in-new-tab-or-window

Reply:
done